astro/src/styles/bootstrap.scss

```diff
@@ -64,17 +64,34 @@ $custom-colors: (
   "ac": #041058,
   "ac2": #95B2E0,
   "ac3": #A9B6FB,
+  "light-secondary": #fcfeff,
+  "light-tertiary": #eff0f3,
+  "dark-secondary": #232635,
+  "dark-tertiary": #6C717E,
   "facebook": #4267B2,
   "instagram": #5B51D8,
   "linkedin": #2E64BC,
   "mastodon": #6364FF,
   "paypal": #253B80, // Dark cornflower blue
   "stripe": #625BF6, // Stripe Blurple
-  "threads": black,
+  "threads": #000000,
   "twitter": #1DA1F2,
 );
 $theme-colors: map-merge($theme-colors, $custom-colors);
 
+// Define background colors for light mode
+$body-secondary-bg:                 #FCFEFF;
```

Comment:
The color you've chosen here is so light that's indiscernible from the default `$body-bg` which is white. Let's have a slightly darker shading on secondary.